prow/config/external_plugins_config.yaml

```diff
@@ -555,26 +555,17 @@ ti-community-label:
       - subject
       - type
     additional_labels:
-      - 'bug-from-poc'
-      - 'bug-from-user'
-      - 'contribution-welcome'
-      - 'dba-test'
-      - 'dependencies'
-      - 'documentation'
       - 'duplicate'
-      - 'enhancement'
-      - 'feature-request'
-      - 'good-first-issue'
       - 'hacktoberfest'
```

Comment:
Should we keep this, it's a community activity, which has ended. (In fact no one took part in.

Reply:
Deleted.